marklogic-data-hub-central/ui/src/components/modeling/property-table/property-table.test.tsx

```diff
@@ -244,6 +244,10 @@ describe('Entity Modeling Property Table Component', () => {
     fireEvent.change(facetableCheckbox, { target: { checked: true } });
```

Comment:
Could you update the mock entity model in "propertyTableEntities" with the changes you made to the Customer entity? This way you could test for assertions of 'sortable' and 'facetable' in the Property table.

Reply:
Going to leave out adding sortable properties in this PR as sortable is not yet merged (https://github.com/marklogic/marklogic-data-hub/pull/4230). When that feature is merged, the PR has to take care of it. @rahulvudutala will add sortable properties at a later time.